R/nested_closeable_modal.R

```diff
@@ -0,0 +1,99 @@
+#' Nested Closeable modal popup
+#'
+#' @description `r lifecycle::badge("experimental")`
+#' Alternative to `shiny::modalDialog``. To create a nested modal popup that can be shown/hidden
+#' using JS using the modal `id` without disturbing the parent modal
```

Comment:
```suggestion
#' Alternative to `shiny::modalDialog`. Create a nested modal popup that can be shown/hidden
#' using JS and modal `id`, without disturbing the parent modal.
```

Reply:
Ah! I just pushed the change for `. But thanks for your 🦅 eyes